packages/wdio-types/src/Capabilities.ts

```diff
@@ -817,27 +817,398 @@ export interface IECapabilities {
  * see also https://docs.saucelabs.com/dev/test-configuration-options
  */
 export interface SauceLabsCapabilities {
+    /**
+     * Desktop Browser Capabilities: Sauce-Specific – Optional
+     * see https://docs.saucelabs.com/dev/test-configuration-options/#desktop-browser-capabilities-sauce-specific--optional
+     */
+
+    /**
+     * Allows you to specify the ChromeDriver version you want to use for your tests. 
+     * The default version of ChromeDriver when no value is specified depends on the version of Chrome used. 
+     * As of Chrome 73, the major version of the driver and the browser must match.
+     * 
+     * VDC only.
```

Comment:
Many people probably won't understand this acronym, can we write this out?

Reply:
Absolutely, I can change that.

Reply:
Added another push to replace the acronym